r-package/survlearners/R/surv_fl_grf.R

```diff
@@ -69,6 +69,9 @@ surv_fl_grf <- function(X, Y, W, D, t0, W.hat = NULL, cen.fit = "Kaplan-Meier",
     cen.times.index <- findInterval(U, c.fit$failure.times)
     C.hat <- C.hat[cbind(1:length(U), cen.times.index)]
   }
+  if (any(C.hat == 0)) {
+    stop("Some or all uncensored probabilities are exactly zeros. Check input variables or consider adjust the time of interest t0.")
```

Comment:
I don't understand what "check input variables" is supposed to mean

Reply:
The input variable Y may be so large due to using the inconsistent units, e.g., Y is in days but t0 is in years

Reply:
Say when the units of the input variables are inconsistent:  Y is in years but t0 is in days